funnel/templates/proposal.html.jinja2

```diff
@@ -118,20 +118,17 @@
     <section class="proposal__section--border">
 
       <div class="proposal__admin-panel sticky-admin-panel">
-        {%- if proposal.current_roles.owner or proposal.current_roles.admin %}
+        {%- if proposal.current_roles.project_editor %}
           <div class="card">
             <div class="card__header">
               <h3 class="mui--text-title"><strong>ADMIN PANEL</strong></h3>
             </div>
             <div class="card__body">
-              {%- if proposal.current_roles.owner %}
                 <ul class="mui-list--aligned">
                   <li><a href="{{ proposal.url_for('edit') }}">{% trans %}Edit details{% endtrans %}</a></li>
                   <li><a href="{{ proposal.url_for('delete') }}">{% trans %}Delete{% endtrans %}</a></li>
                 </ul>
                 <hr class="mui-divider">
-              {%- endif %}
-              {% if proposal.current_roles.admin %}
```

Comment:
Proposals don't have an 'admin' role. However, proposals will inherit the `project_editor` role from the project.